src/middleware/types.ts

```diff
@@ -200,14 +321,15 @@ export interface MessageAction extends KeyValueMapping {
     enterprise_name?: string; // undocumented
   };
   token: string;
-  action_ts: string; // undocumented
```

Comment:
have we checked that this really isn't present? i thought i manually did when i added it.